src/qt/region.cpp

```diff
@@ -106,7 +106,7 @@ wxRegion::wxRegion(const wxBitmap& bmp, const wxColour& transp, int tolerance)
         return;
     }
 
-    unsigned char raw[bmp.GetWidth()*bmp.GetHeight()];
+    unsigned char* raw = new unsigned char(bmp.GetWidth()*bmp.GetHeight());
```

Comment:
Did you by any chance mean `[]` here? Considering the `memset` below, did you test this change?
What was wrong with the original code? Was the mem chunk too big for the stack?

Reply:
Its length was not variable because it didn't change later. The old size was not a *compile time constant* which was indeed a problem. But you do need the square brackets, otherwise you get a pointer to only one char, initialized to `bmp.GetWidth()*bmp.GetHeight()` value, don't you?

Reply:
Yes, thank you for noticing. I have fixed it.

Reply:
The best would be to use `wxScopedArray<unsigned char> raw(bmp.GetWidth()*bmp.GetHeight())` here.